akka-actor/src/main/scala-2.13-/akka/util/ccompat/ImmutableSortedMapExtensions.scala

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.util.ccompat
+
+import scala.collection.{ immutable ⇒ i }
+
+class ImmutableSortedMapExtensions(private val fact: i.SortedMap.type) extends AnyVal {
```

Comment:
does it have to be public?

Reply:
I think it has to be visible to the class where the extensions are used, so I think it can be `private[akka]` but not more.